build/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM golang:1.19.0 AS go_builder
 WORKDIR /go/src/github.com/percona/percona-server-mysql-operator
 
-COPY . .
```

Comment:
@egegunes I do not see any time build differences  with this line and without :(